master/getting-started/openstack/upgrade/downgrade.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Downgrading Calico
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/openstack/upgrade/downgrade'
+---
+
+## About downgrading {{site.prodname}}
+
+Under some circumstances, you may need to perform a downgrade and return your
+cluster to the previous version of {{site.prodname}}. If you need to downgrade
+you should do so as soon as possible to avoid an outage.
+
+> **Important**: After downgrading or aborting the migration it is necessary
+> to delete the previously migrated
+> [etcd](delete#deleting-calico-data-from-etcdv2-after-a-successful-migration-and-upgrade)
```

Comment:
I think that you might need to direct them to the section about deleting calico data after a _partial_ migration.